frontend/common/src/intl/locales/en/system.json

```
@@ -31,6 +32,7 @@
   "label.add-route": "Add Route",
   "label.add-strength": "Add Strength",
   "label.add-unit": "Add Unit",
+  "label.approve-next": "Approve & Next",
   "label.brand": "Brand",
   "label.brands": "Brands",
   "label.component": "Comoponent",
```

```suggestion
  "label.component": "Component",
```

Not from this PR, but noticed it here :)